### PR TITLE
Bug: Add Trust name to the Trust Involved Page

### PR DESCRIPTION
**Pull request by SteveHicksCH**
### JIRA link
[ROE-1286](https://companieshouse.atlassian.net/browse/ROE-1286)

### Change description
Get Trust name from session for current trust

### Work checklist

- [x] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by JoshFen95 — APPROVED** [review]
looks good to me

File: src/utils/trust/who.Is.Involved.mapper.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import * as Trust from 'model/trust.model';
```
Comment (markpit) [inline src/utils/trust/who.Is.Involved.mapper.ts:1]: file name needs to be in lower case
 Is.Involved - > is.involved

File: src/utils/trust/who.Is.Involved.mapper.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { TrustWhoIsInvolved } from 'model/trust.page.model';
+
+const mapTrustWhoIsInvolvedToPage = (
```
Comment (markpit) [inline src/utils/trust/who.Is.Involved.mapper.ts:4]: the other utils/trust/mapper modules have their own unit test, would you be able to add one for this too.

File: src/utils/trust/who.Is.Involved.mapper.ts (hunk 1 above)
Comment (SteveHicksCH) [reply src/utils/trust/who.Is.Involved.mapper.ts]: @markpit - File renamed

File: src/utils/trust/who.Is.Involved.mapper.ts (hunk 2 above)
Comment (JoshFen95) [reply src/utils/trust/who.Is.Involved.mapper.ts]: Good point. Added the unit test

**Review by dgolubevs — APPROVED** [review]

**Pull request merged**